Tests/SwiftPrettyPrintTests/Public/CombineExtensionTests.swift

```diff
@@ -40,6 +40,83 @@ final class CombineExtensionTests: XCTestCase {
         [3, 4],
     ]
     
+    func testPrefix() throws {
+        let recorder = StringRecorder()
+        let exp = expectation(description: "")
+        
+        array
+            .publisher
+            .prettyPrint("🍎", format: .singleline, to: recorder)
+            .handleEvents(receiveCompletion: { _ in
+                exp.fulfill()
+            })
+            .sink { value in }
+            .store(in: &cancellables)
+        
+        wait(for: [exp], timeout: 3)
+        
+        // Note:
+        // Order of first and second line is unstable.
+        XCTAssert(recorder.contents.contains("🍎: receive subscription: [[1, 2], [3, 4]]"))
+        XCTAssert(recorder.contents.contains("🍎: request unlimited"))
+        assertEqualLines(recorder.contents.split(separator: "\n")[2...].joined(separator: "\n"),
+            """
+            🍎: receive value: [1, 2]
+            🍎: receive value: [3, 4]
+            🍎: receive finished
+            """)
+    }
+    
+    func testWhen() throws {
```

Comment:
I don't test to all patterns because obvious implementation that call local function `_print()` and pass to explicit argument `type`.
https://github.com/YusukeHosonuma/SwiftPrettyPrint/pull/120/files#diff-d0614fd28472d63624012ad75ee56f32a5cc3c2c35cefadbc1035e2c03f366dfR38